hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -405,6 +408,73 @@ public void testNoHeaders() throws IOException {
         }
     }
 
+    @Test
+    public void testMapGetWithEscapedName() throws IOException {
+        String mapName = randomMapName();
+        IMap<String, String> map = instance.getMap(mapName + " a");
+        map.put("key 1", "value1");
+        map.put("key2", "value2");
+        map.put("key3", "value3");
```

Comment:
minor: I believe these puts don't add anything to the test logic

Reply:
Removed